FeedStoreChallenge/CD_FeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CD_FeedImage)
+public class CD_FeedImage: NSManagedObject {}
```

Comment:
`CD_FeedImage` is an internal implementation detail of `CoreDataFeedStore`, thus it should not be public.
Also this class is not intended to be inherited thus it should be `final`
Please remove all `public` modifiers from this class declaration and its properties and methods and make the class `final`.

```suggestion
final class CD_FeedImage: NSManagedObject {}
```